### ISSUE TITLE
npm and docker results are not the same

### ISSUE THREAD

**Issue by sinkcup** [nvuillam/npm-groovy-lint#80]
```Jenkinsfile
pipeline {
  agent any
  stages {
              stage('test') {
      steps {
echo 'bad indent'
      }
}
}
```

```shell
npx npm-groovy-lint -f Jenkinsfile
# 0 error, 0 warning, 0 info
```

```shell
docker run -u "$(id -u):$(id -g)" -w=/tmp -v "$PWD/Jenkinsfile":/tmp/Jenkinsfile nvuillam/npm-groovy-lint
```

```
GroovyLint: Started CodeNarc Server
/tmp/Jenkinsfile
  1     warning  Class should be marked with one of @GrailsCompileStatic, @CompileStatic or @CompileDynamic  CompileStatic           
  2     info     The statement on line 2 in class None is at the incorrect indent level: Expected column 1 but was 3  Indentation             
  3     info     The statement on line 3 in class None is at the incorrect indent level: Expected column 1 but was 3  Indentation             
  4     info     The statement on line 4 in class None is at the incorrect indent level: Expected one of columns [5, 9, 13] but was 15  Indentation             
  5     info     The statement on line 5 in class None is at the incorrect indent level: Expected one of columns [9, 13, 17] but was 7  Indentation             
  6     info     The statement on line 6 in class None is at the incorrect indent level: Expected one of columns [13, 17, 21] but was 1  Indentation             

/tmp/npm-groovy-lint/codeNarcTmpRs_0.5641933801119414.groovy
  1     warning  Class should be marked with one of @GrailsCompileStatic, @CompileStatic or @CompileDynamic  CompileStatic           
  2     info     Code block starts with a blank line.  BlockStartsWithBlankLine
  159   info     The map entry spacesPerIndentLevel:4 within class None is not preceded by a space or whitespace  SpaceAfterComma         
  359   info     Code block ends with a blank line.  BlockEndsWithBlankLine  


npm-groovy-lint results in 2 linted files:
┌─────────┬───────────┬─────────────┐
│ (index) │ Severity  │ Total found │
├─────────┼───────────┼─────────────┤
│    0    │  'Error'  │      0      │
│    1    │ 'Warning' │      2      │
│    2    │  'Info'   │      8      │
└─────────┴───────────┴─────────────┘
package.json not found, use default value {"name":"npm-groovy-lint","version":"0.0.0"} instead
```

their results are not the same, and docker output two files but I only mount 1 file to it.

**Comment by nvuillam** [comment]
It's because your test cases are in /tmp folder ^^

As CodeNarc requires [RuleSet files in groovy format](https://codenarc.github.io/CodeNarc/codenarc-creating-ruleset.html), I generate them "on the fly" from groovylintrc.js analysis , then store them in a file that I send to CodeNarc as argument

The second file you see is this temporary file ( stored in /tmp, like your test file )
I think that if you put your file anywhere else , you will have a single result :)

Note: in the future, I'll propose a PR to CodeNarc where I can send the RuleSets content as argument instead of file, so this issue should disappear 😇

**Comment by nvuillam** [comment]
Addendum: seems also that your npx call has an issue 

Try  with Ant-Style arguments :)

`npx npm-groovy-lint -f **/Jenkinsfile`

![image](https://user-images.githubusercontent.com/17500430/89260781-3fc57900-d62d-11ea-86bd-a8734f138e3d.png)

**Comment by sinkcup** [comment]
this project only has 1 Jenkinsfile, it's `./Jenkinsfile`, if use `**/Jenkinsfile`, it will check `./node_moudles/xxx/Jenkinsfile`, so how can I check `./Jenkinsfile`

**Comment by nvuillam** [comment]
hmmmm there must be some Ant way to write it 😄 

maybe `--files */Jenkinsfile` ?

https://ant.apache.org/manual/dirtasks.html#patterns

**Comment by sinkcup** [comment]
I have tried `*/Jenkinsfile`, it's wrong.
ignore maybe OK, but it's not easy to use

**Comment by nvuillam** [comment]
`./Jenkinsfile` ?

**Comment by sinkcup** [comment]
`./Jenkinsfile` is same as `Jenkinsfile`, output nothing.

**Comment by nvuillam** [comment]
hmmmm i'm struggling but i didn't find a solution yet
my real job is starting, i may check later ;)
